.gitignore

```diff
@@ -0,0 +1,12 @@
+
+boostedhiggs/processors/.DS_Store
```

Comment:
simply include *.DS_Store so that it ignores all possible occurrences. Then delete the one inside `boostedhiggs`